options.js

```diff
@@ -46,25 +46,27 @@ function httpOptions (yargs) {
             alias: 'host',
             demandOption: false,
             default: config.elasticHost,
-            describe: 'ElasticSearch URL'
+            describe: 'ElasticSearch URL',
         })
         .option('p', {
             alias: 'port',
             demandOption: false,
             default: config.port,
             describe: 'The port to run the auth code server on'
         })
-        .option('p', {
+        .option('r', {
```

Comment:
Could we put this in the file?  Like a file that would give the count, schema, HTTP verb (POST/PUT) and route

```js
{
    count: 100,
    route: /v1/source/bla,
    httpVerb: "PUT",
    shema: <schema>
}
```

If we have a lot of paths, seems like that would be a lot to type.